src/workstation-gui/app/javascript/controllers/hello_controller.js

```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    this.element.textContent = "Hello World!"
+  }
+}
```

Comment:
we will have to get rid of these once the app is stable as frontend files and libs may be heavy while binding as well.